chain-abci/src/app/rewards.rs

```diff
@@ -155,15 +142,20 @@ mod tests {
         let state = app.last_state.as_ref().unwrap();
         let top_level = &state.top_level;
         let reward1 = monetary_expansion(
+            total_staking.into(),
             top_level.rewards_pool.tau,
-            total_staking,
-            top_level.rewards_pool.minted,
-            &top_level.network_params,
+            top_level
+                .network_params
+                .get_rewards_monetary_expansion_r0()
+                .as_millis(),
+            top_level.network_params.get_rewards_reward_period_seconds() as u64,
         );
+        println!("reward1: {}", reward1);
+        let reward1 = Coin::new(reward1).unwrap();
         let mut req = env.req_begin_block(1, 0);
         req.mut_header().set_time(seconds_to_timestamp(
             state.block_time
-                + top_level.network_params.get_rewards_distribution_period() as Timespec,
+                + top_level.network_params.get_rewards_reward_period_seconds() as Timespec,
```

Comment:
since , when some comments?

Reply:
What do you mean?

Reply:
simple  explanation of "get_rewards_reward_period_seconds"